content/en/users/cloud-compute/storage/_index.md

```diff
@@ -86,3 +86,61 @@ Once you have a filesystem you can mount it at the desired path:
 With that you can access `/<path>` where all your data will be available.
 Applications will not see any difference between a block storage device and a
 regular disk, thus no major changes should be required in the application logic.
+
+## Exposing storage to EGI File Transfer service
+
+The [File Transfer service](../../data-transfer) allows you to move any type of
+data files asynchronously from one storage to another. If you want to copy data
+from/to one VM running on the EGI cloud, you will need to run a compatible
+server (Webdav/https, GridFTP, xrootd, SRM, S3, GCloud) that can interact with
+the FTS3 software.
+
+An easy way to provide a GridFTP server on your VM is to use the
+[gridftp-le ready2go docker stack](https://github.com/cern-fts/ready2go/tree/master/gridftp-le)
+for deploying a GridFTP Docker Container with certificates from Let's Encrypt.
+Take into account:
+
+- Security groups for the VM must allow ports 80, 2811 and the 50000-50200 range.
+- The VM must have a valid DNS entry (you can use
+  [FedCloud's Dynamic DNS](https://nsupdate.fedcloud.eu) for getting one)
+- The default setup uses `/srv` as path to expose and maps users to the `nobody`
+  user. Make sure that `nobody` is able to read (and write if needed) on that
+  location or set the mapping to the appropriate users.
+- The Let's Encrypt certificates may not be accepted by some the EGI
+  infrastructure endpoints, you may want to consider using IGTF certificates
+  instead. Check your CA for instructions on how to get those.
+- You can add direct mappings for specific DNs by adding a
+  `/etc/localgridmap.conf` file in your running container. See the example below
+  to map the
+  `/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Enol Fernandez del Castillo`
+  DN to `nobody`. You can add as many lines as needed:
+
+  <!-- markdownlint-disable line-length -->
+  ```plaintext
+  "/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Enol Fernandez del Castillo" nobody
+  ```
+  <!-- markdownlint-enable line-length -->
+
+- You need to specify the environment variables in the `docker-compose.yml` file
+  for obtaining the Let's Encrypt certificate. An extra variable
+  `GLOBUS_HOSTNAME` must be also set:
+
+  ```yaml
+  environment:
+    - TESTCERT
+    - EMAIL=youremail@domain.com
+    - DOMAIN=mygridftp.example.com
+    - GLOBUS_HOSTNAME=mygridftp.example.com
```

Comment:
is GLOBUS_HOSTNAME also needed? on the code i see that GLOBUS_HOSTNAME is set the value of DOMAIN https://github.com/cern-fts/ready2go/blob/master/gridftp-le/Dockerfile#L30

Reply:
actually i think there is a but there , as it should be `ENV GLOBUS_HOSTNAME=$DOMAIN`

Reply:
it's needed, as the Dockerfile sets the value but that's container build time, on runtime if it was not properly set before, you will still need it